contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -120,47 +70,43 @@ def _clone_repo(repo_url, verifyssl=True, branch='master'):
         Repo.clone_from(repo_url, abs_local_path, branch=branch, depth=1)
         return abs_local_path
 
-    def _move_packs(self, abs_repo_base, packs, abs_local_path, subtree):
+    def _move_pack(self, abs_repo_base, pack_name, abs_local_path):
```

Comment:
We can either check for the dir and abort the install if it's not empty, as @enykeev says (a sensible approach), or update the pack and keep the config (that's what we do now, and that's why we have temp clone). I'd go for the latter, at least for now—makes things easier for user, and even a bit easier for us.